cmd/parse.go

```diff
@@ -77,5 +105,7 @@ func parse(args []string, stdout io.Writer, stderr io.Writer) int {
 
 func init() {
 	parseCommand.Flags().VarP(parseParams.format, "format", "f", "set output format")
+	parseCommand.Flags().StringVarP(&parseParams.jsonInclude, "json-include", "", "", "select optional elements, current options: locations, comments. E.g. --json-include locations,-comments will include locations and exclude comments.")
```

Comment:
I'm curious why `comments` was  included as a `--json-include` option. Is it because it's easy to remove from the parsed module or something else?

Reply:
That's a good question. I added it for two reasons:

* Comments use an inconsistent JSON key name where the fields are capitalised for legacy reasons. For this reason, a use of the command might want to drop them if they are interested in walking the JSON and processing it in an automated way.
* I wanted to include an additional option beyond locations for this flag so that it was clear how it was to be used.

Clearly the latter reason isn't very strong and I'd be more than happy to remove this functionality. I just thought it make it clearer in the PR how that flag was used.